localization-pack-es/StringTableEsUI.xml

```diff
@@ -293,7 +293,7 @@
             <GameDBLocalizedString>      <LocID>NUMBER_OF_AMBULANCES</LocID>            <Text>Compra {1} ambulancia(s)</Text>  </GameDBLocalizedString>
             <GameDBLocalizedString>      <LocID>BALANCE</LocID>                         <Text>Ten {1} en tu cuenta</Text>  </GameDBLocalizedString>
             <GameDBLocalizedString>      <LocID>BANK_BALANCE</LocID>                    <Text>Ten un saldo mayor que cero</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>PROFIT_PER_DAY</LocID>                  <Text>Obtén beneficio durante {1} día(s) consecutivo/s.</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>PROFITABLE_DAYS</LocID>                  <Text>Obtén beneficio durante {1} día(s) consecutivo/s.</Text>  </GameDBLocalizedString>
```

Comment:
Is this intentional? Why is the LocID changed here? I'm afraid of accepting the PR because of this.
Did the LocID change, @jan-oxymoron ?

Reply:
Hi guys, i think this was an error, both lines should be in place, i mean both LocID should be in the file afaik.